org.eclipse.iot.fourdiac.Ide.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.eclipse.iot.fourdiac.Ide",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```

Comment:
This should change to the latest release, 18.08 which now has an OpenJDK10 release and is the only runtime that OpenJDK 11 will be available for.

Reply:
Maybe someone should update the docs ;-) I updated to `18.08`, that looks good.